diff --git a/ak3/anykernel.sh b/ak3/anykernel.sh
index 4b72972d4d..2a41fa3b1d 100755
--- a/ak3/anykernel.sh
+++ b/ak3/anykernel.sh
@@ -79,18 +79,4 @@ write_boot; # use flash_boot to skip ramdisk repack, e.g. for devices with init_
 ## end vendor_kernel_boot install
 
 
-## vendor_boot shell variables
-block=vendor_boot;
-is_slot_device=1;
-ramdisk_compression=auto;
-patch_vbmeta_flag=auto;
-
-# reset for vendor_boot patching
-reset_ak;
-
-# vendor_boot install
-dump_boot; # use split_boot to skip ramdisk unpack, e.g. for dtb on devices with hdr v4 but no vendor_kernel_boot
-
-write_boot; # use flash_boot to skip ramdisk repack, e.g. for dtb on devices with hdr v4 but no vendor_kernel_boot
-## end vendor_boot install
-
+## vendor_boot: do not repack on OP9, only replace kernel in boot
